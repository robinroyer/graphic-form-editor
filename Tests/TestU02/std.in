R r01 10 10 5 5
DELETE r01
UNDO
REDO
LIST
UNDO
LIST
EXIT