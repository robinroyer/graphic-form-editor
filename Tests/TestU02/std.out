OK
#New object: r01
OK
#Objects sucsessfully removed
OK
#Last command successfully 'undo'ed
OK
#Last command successfully 'redo'ed
OK
#Last command successfully 'undo'ed
R r01 10 10 5 5
